app/src/main/java/kth/jjve/xfran/adapters/WorkoutsRecyclerAdapter.java

```diff
@@ -0,0 +1,107 @@
+package kth.jjve.xfran.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import kth.jjve.xfran.R;
+import kth.jjve.xfran.models.Workout;
+
+/**
+ * Adapter for Workouts tab
+ */
+
+public class WorkoutsRecyclerAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+
+    private final String LOG_TAG = getClass().getSimpleName();
+    private List<Workout> mWorkouts = new ArrayList<>();
+    private ArrayList<Boolean> mExpandedStatus = new ArrayList<>();
+    private Context mContext;
+    final private ListItemClickListener mOnClickListener;
```

Comment:
make sure you use the same order to keep the code neat. So private final instead of final private